alchemist-graphql/src/commonMain/kotlin/it/unibo/alchemist/boundary/graphql/client/DefaultGraphQLClient.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.boundary.graphql.client
+
+import com.apollographql.apollo3.ApolloCall
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.api.Mutation
+import com.apollographql.apollo3.api.Query
+import com.apollographql.apollo3.api.Subscription
+import com.apollographql.apollo3.network.ws.GraphQLWsProtocol
+import com.apollographql.apollo3.network.ws.WebSocketNetworkTransport
+
+/**
+ * Default GraphQL client implementation.
+ *
+ * @param host the host of the GraphQL server
+ * @param port the port of the GraphQL server
+ * @param enableSubscription whether to enable subscriptions or not
+ */
+class DefaultGraphQLClient(
+    private val host: String = "127.0.0.1",
+    private val port: Int = 8081,
```

Comment:
This number occurs too many times. maybe create a constant in a (companion?) object